clients/src/test/java/org/apache/kafka/common/metrics/stats/FrequenciesTest.java

```diff
@@ -46,7 +46,7 @@
     public void setup() {
         config = new MetricConfig().eventWindow(50).samples(2);
         time = new MockTime();
-        metrics = new Metrics(config, Arrays.asList((MetricsReporter) new JmxReporter()), time, true);
```

Comment:
unused import checkstyle error.

Reply:
Thanks, I had forgotten to push the updated branch after I fixed this.